addOns/pscanrulesAlpha/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 - Add Java Serialized Object (JSO) Scanner.
 
+## [26] - 2019-08-15
```

Comment:
This line (and empty lines around) should be removed.

Reply:
done changes as suggested.